yarn.lock

```diff
@@ -3494,6 +3494,11 @@ posix-character-classes@^0.1.0:
   resolved "https://registry.yarnpkg.com/posix-character-classes/-/posix-character-classes-0.1.1.tgz#01eac0fe3b5af71a2a6c02feabb8c1fef7e00eab"
   integrity sha1-AerA/jta9xoqbAL+q7jB/vfgDqs=
 
+prettier@^2.4.1:
```

Comment:
Can't match version in `package.json`:  `2.1.1`

Reply:
ran yarn osd bootstrap again and yarn.lock is updated

Reply:
after updating it looks like this: 
first line changed to 2.1.1, not sure why line below still says version "2.4.1"
```
prettier@^2.1.1:
  version "2.4.1"
  resolved "https://registry.yarnpkg.com/prettier/-/prettier-2.4.1.tgz#671e11c89c14a4cfc876ce564106c4a6726c9f5c"....
```

Reply:
^ usually if yarn finds an existing dependency in `yarn.lock` and it is later upgraded, it will keep the existing alias if used by other dependencies. For example, some other dependency may need `prettier@^2.1.1` (AKA some version of prettier >= v2.1.1). The underlying version `2.4.1` is what's actually referenced and used here, as you can see by the link to the `prettier` tarball in the yarn package registry